config/initializers/assets.rb

```diff
@@ -15,4 +15,4 @@
 Rails.application.config.assets.precompile += %w( application.css application.js tylium.js tylium.css bootstrap.css)
 
 # Include font files
-Rails.application.config.assets.precompile += [/\.(?:svg|eot|woff|ttf)\z/]
+Rails.application.config.assets.precompile << ["*.svg", "*.eot", "*.woff", "*.ttf"]
```

Comment:
as far as I can tell, we don't use .woff, or .eot; not even .svg...

Reply:
Font awesome uses those formats so this is likely for vendor compatibility. We do package them in our assets.

<img width="610" alt="image" src="https://user-images.githubusercontent.com/179134/105357982-db579f80-5bf5-11eb-85bf-ab7bd6b33c88.png">